Bucketeer/Sources/Internal/Event/EventInteractor.swift

```diff
@@ -407,8 +407,18 @@ extension BKTError {
                 )
             )
             metricsEventType = .unknownError
-        case .unknown:
-            metricsEventData = .unknownError(.init(apiId: apiId, labels: labels))
+        case .unknown(let message, _):
+            metricsEventData = .unknownError(
+                .init(
+                    apiId: apiId,
+                    labels: labels.merging(
```

Comment:
I think the `tag` is also missing in the metric event labels.
Can you confirm?

Reply:
The tag is inserted here https://github.com/bucketeer-io/ios-client-sdk/blob/654dbba0b55d15a57d1ec910a4851c66b58ed880/Bucketeer/Sources/Internal/Event/EventInteractor.swift#L173

and we are using merging() to merge new changes to the input map